src/Sylius/Bundle/UserBundle/Resources/config/services.xml

```diff
@@ -58,6 +58,11 @@
 
         <parameter key="sylius.user.validator.unique_registered_user.class">Sylius\Bundle\UserBundle\Validator\Constraints\RegisteredUserValidator</parameter>
         <parameter key="sylius.user.validator.customer_initializer.class">Sylius\Bundle\UserBundle\Validator\Initializer\CustomerInitializer</parameter>
+
+        <parameter key="sylius.form.type.user_change_password.date_class">Sylius\Bundle\UserBundle\Form\Model\ChangePassword</parameter>
```

Comment:
Still, every class has its parameter under sylius.some_resource.class. And this parameter is injected to every resource type.